DeviceAdministration/Infrastructure/BusinessLogic/DeviceLogic.cs

```diff
@@ -1143,8 +1144,8 @@ public DeviceListLocationsModel ExtractLocationsData(List<DeviceModel> devices)
 
                     try
                     {
-                        latitude = DeviceSchemaHelper.GetDeviceProperties(device).Latitude.Value;
-                        longitude = DeviceSchemaHelper.GetDeviceProperties(device).Longitude.Value;
+                        latitude = device.DeviceProperties.Latitude.Value;
```

Comment:
Yes. DeviceProperties.Lattitue is a Nullable<double> and the value being assigned is a double. So need to call .Value to unwrap the nullable and get the double out.